### DISCUSSION TITLE
track ticket selling

### DISCUSSION THREAD

**Discussion by alemao8** [pretix/pretix#4283] [Q&A] [answered]
Hello !
Is it possible to track the purchases of tickets for specific products and events in pretix by using Google Analytics platform?
Is this possible in a small size installation in a customer's premises ?

**Comment by pc-coholic** [comment, marked as answer]
Not out of the box.

There is a [plugin](https://marketplace.pretix.eu/products/tracking/) available, which provides such functionality.

It is included on pretix.eu Hosted Service and can be licensed for a fee for self-hosting environments.

**Reply by alemao8** [reply]
Thank you pc-coholic.
I can see from the screenshot that exists in the plugin page you sent me, that the only configuration for Google Analytics is to place the tracking ID. Is this enough from pretix's side, in order to track the purchases of products and events in pretix ?
Is there a documentation for this ?

**Reply by wiffbi** [reply]
Yes, for Google Analytics this is enough. It will track pageviews and purchase as well as begin_checkout, add_to_cart (experimental though), add_shipping_info, add_payment_info (last one might go away depending on security requirements on payment pages).

I am sorry, there is no further documentation on that plugin.

**Reply by alemao8** [reply]
Could you provide me some more technical details?
For example will a php/javascript snippet added to the corresponding pages of pretix ? Do I have to place any code in pretix or all the configuration will come from Google Analytics ?

Thank you in advance

**Reply by wiffbi** [reply]
When you add your measurement ID to the settings in pretix, that plugin will add a javascript that loads the gtag.js from Google and automatically adds all the relevant information to the `window.dataLayer`, which then is tracked as usual from Google Analytics. You do not need nor can’t change anything – just the measurement ID in the settings.

**Reply by alemao8** [reply]
Thanx for the clarification wiffbi. This is very useful

**Comment by alemao8** [comment]
Can I use this plugin mentioned above to track ticket purchases from Facebook too and any other known social media ?

**Reply by pc-coholic** [reply]
The tracking plugin will track everything to your chosen platform (GTM, Facebook Pixel, ...), independently of the source.
